src/track/trackmetadatataglib.cpp

```diff
@@ -232,6 +232,71 @@ bool parseBpm(TrackMetadata* pTrackMetadata, QString sBpm) {
     return isBpmValid;
 }
 
+void parseRating(TrackMetadata *pTrackMetadata, QString sRating, FileType filetype = FileType::UNKNOWN) {
```

Comment:
Always prefer to write functions with a result and no side effects over functions with no result but with side effects.